src/index.js

```diff
@@ -1,28 +1,52 @@
 import './index.css';
+import {
+  createGameScores,
+  getGameScores,
+  saveGameScore,
+} from '../modules/gameFunc.js';
 
-const submit = document.querySelector('.submit');
-const nameInput = document.getElementById('name');
-const scoreInput = document.getElementById('score');
-const scoresList = document.getElementById('scores');
+// 1. Create a new game and save its ID
+// 2. Add functionalities for refreshing and submitting the game
 
-// create a new score item for the list
-const createScores = (name, score) => {
-  const li = document.createElement('li');
-  li.textContent = `${name}: ${score}`;
-  return li;
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+let gameId;
+const createGame = async () => {
+  const response = await fetch(`${apiUrl}games/`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: 'My Game' }),
+  });
+  const data = await response.json();
+  gameId = data.result.slice(-5);
 };
+createGame();
 
-submit.addEventListener('click', (e) => {
-  e.preventDefault();
+// Refresh button
+const refreshButton = document.querySelector('.refresh');
+const scoresList = document.getElementById('scores');
+refreshButton.addEventListener('click', async () => {
+  const scores = await getGameScores(gameId);
+  scoresList.innerHTML = '';
+  scores.forEach((score) => {
+    const li = createGameScores(score.user, score.score);
+    scoresList.appendChild(li);
+  });
+});
 
+// Submit button
+const submitButton = document.querySelector('.submit');
+const nameInput = document.getElementById('name');
+const scoreInput = document.getElementById('score');
+submitButton.addEventListener('click', async (e) => {
+  e.preventDefault();
   const name = nameInput.value;
   const score = scoreInput.value;
-
-  // Add new score to the list
-  const newScore = createScores(name, score);
+  await saveGameScore(gameId, name, score);
+  const newScore = createGameScores(name, score);
   scoresList.appendChild(newScore);
```

Comment:
- Well done,  but according to the project requirements, `The submit button should hit the POST API endpoint and post user entries` and `When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change`. Please ensure that your submit function only submits the scores, the scores list should change when the refresh button is clicked. Please fix this 😄